src/components/form/FormFieldDescription.tsx

```diff
@@ -29,6 +29,9 @@ interface Props {
   validation?: `${FormFieldValidation}`;
 }
 
+// TODO do we need aria-live="polite" ?
```

Comment:
Good point. I found this [guide](https://www.smashingmagazine.com/2023/02/guide-accessible-form-validation/#invalid-fields) but I'm running out of time for today. I will revisit it tomorrow.